From 4b4a4c80b06ab8ea76ffeac346bad46d9087fcdf Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?S=C3=A9rgio=20M=2E=20Basto?= <sergio@serjux.com>
Date: Sat, 16 Nov 2019 04:26:07 +0000
Subject: [PATCH 5/5] Fedora 30 Qt Fixup for

filesavedialog.h:7:10: fatal error: private/qguiapplication_p.h: No such file or directory

I had to disable HELP dir
---
 pteid-mw-pt/_src/eidmw/eidguiV2/filesavedialog.cpp | 8 --------
 pteid-mw-pt/_src/eidmw/eidguiV2/filesavedialog.h   | 1 -
 2 files changed, 9 deletions(-)

diff --git a/pteid-mw-pt/_src/eidmw/eidguiV2/filesavedialog.cpp b/pteid-mw-pt/_src/eidmw/eidguiV2/filesavedialog.cpp
index 4d7d3035..35a62346 100644
--- a/pteid-mw-pt/_src/eidmw/eidguiV2/filesavedialog.cpp
+++ b/pteid-mw-pt/_src/eidmw/eidguiV2/filesavedialog.cpp
@@ -8,7 +8,6 @@
 
 FileSaveDialog::FileSaveDialog(QQuickItem *parent)
     : QQuickItem(parent)
-    , m_dlgHelper(init_helper())
     , m_modality(Qt::WindowModal)
     #if QT_VERSION >= QT_VERSION_CHECK(5, 8, 0)
     , m_options(QFileDialogOptions::create())
@@ -108,13 +107,6 @@ void FileSaveDialog::setNameFilters(QStringList nameFilters)
     }
 }
 
-QPlatformFileDialogHelper* FileSaveDialog::init_helper()
-{
-    return static_cast<QPlatformFileDialogHelper*>(
-                QGuiApplicationPrivate::platformTheme()->createPlatformDialogHelper(QPlatformTheme::FileDialog)
-                );
-}
-
 void FileSaveDialog::open()
 {
     if (!valid()) return;
diff --git a/pteid-mw-pt/_src/eidmw/eidguiV2/filesavedialog.h b/pteid-mw-pt/_src/eidmw/eidguiV2/filesavedialog.h
index 7577d211..0c5416e8 100644
--- a/pteid-mw-pt/_src/eidmw/eidguiV2/filesavedialog.h
+++ b/pteid-mw-pt/_src/eidmw/eidguiV2/filesavedialog.h
@@ -4,7 +4,6 @@
 #include <QQuickItem>
 #include <QFileDialog>
 #include <QUrl>
-#include <private/qguiapplication_p.h>
 #include <QtGui/qpa/qplatformdialoghelper.h>
 #include <QtGui/qpa/qplatformtheme.h>
 
-- 
2.21.0

